packages/pluggableWidgets/range-slider-web/src/ui/RangeSlider.scss

```diff
@@ -1,6 +1,7 @@
 .widget-range-slider {
     padding: 4px 8px;
     margin-bottom: 16px;
+    flex-grow: 1;
```

Comment:
@iobuhov can you communicate with @leonardomendix how to specifically test scenarios where this is relevant? Think it's good to make sure that this does what we want + doesn't interfere in cases where we don't need it.